lib/tests/streamlit/data_frame_proto_test.py

```diff
@@ -253,11 +252,7 @@ def test_marshall_any_array(self):
 
         with pytest.raises(NotImplementedError) as e:
```

Comment:
No change needed, but just wanted to note that when there isn't a conditional on the expected exception string (as now) this type of construct can also be written as:

`with pytest.raises(NotImplementedError, match="^Dtype <U6 not understood.$"): <body>`

Reply:
Oh yeah, thanks for suggesting that.